glide.yaml

```diff
@@ -70,3 +70,13 @@ import:
   version: 335da9dda11408a34b64344f82e9c03779b71673
   repo: https://github.com/openshift/glog.git
   vcs: git
+- package: github.com/stretchr/testify
```

Comment:
Do you think you get away w/o for this case? I'd like to get a chance to have a closer look at the options.